### DISCUSSION TITLE
Only two slides on mobile.

### DISCUSSION THREAD

**Discussion by IvanWala** [nolimits4web/swiper#5601] [Q&A] [answered]
I want display only two slides in mobile. 

[capture.jpg](https://postimg.cc/9rm0K67H)

I try slidesPer view 1/2/3/4, centeredSlide ture/false but i can achieve this effect.

What can I do?

**Comment by Arhell** [comment, marked as answer]
https://swiperjs.com/swiper-api#param-breakpoints

```
320: {
      slidesPerView: 2,
      spaceBetween: 20
    },
```

**Reply by IvanWala** [reply]
Hi have this 

```
{
          parallax: true,
          init: true,
          loop: false,
          grabCursor: true,
          slidesPerView: 5,
          speed: 800,
          centeredSlides: false,
          navigation: {
            nextEl: '.o-swiper-ventajas .swiper .swiper-button-next',
            prevEl: '.o-swiper-ventajas .swiper .swiper-button-prev',
          },
          breakpoints: {
              320: {
                slidesPerView: 2,
                spaceBetween: 20
              },
              768: {
                slidesPerView: 5,
              },
          },
          autoplay: {
              delay: 5000,
          },
      }
```

And doesn't works

[capture2.png](https://postimg.cc/qNQ6wBt6)

**Comment by Arhell** [comment]
What do you want to achieve, why do you need parallax?
for example here https://codesandbox.io/s/frkpbr?file=/index.html

here is slider 1

here examples https://swiperjs.com/demos

**Comment by IvanWala** [comment]
I actualize the assets and it seems to work.

How can I hide the slides beside the navigation? I try to create a wraper outside the swiper but it seems to break layout.

**Reply by Arhell** [reply]
try to use z-index and you can on the wrapper on the left and right through the before and after to make borders
